frontend/src/scenes/session-recordings/player/list/sharedListLogic.ts

```diff
@@ -30,6 +31,12 @@ export const sharedListLogic = kea<sharedListLogicType>([
                 setTab: (_, { tab }) => tab,
             },
         ],
+        showOnlyMatching: [
```

Comment:
I think we should move this to the `playerSettingsLogic` or somewhere that isn't keyed on the recording. That way, you can toggle this setting and then quickly click through recordings in the playlist, and the setting is persisted

Reply:
good idea!